catalog/rest/catalog-rest-service/src/main/java/org/codice/ddf/rest/service/AbstractCatalogService.java

```diff
@@ -372,6 +377,15 @@ public BinaryContent getDocument(
         // We need to catch this exception here or else execution will return to CXF and
```

Comment:
✏️ Does this comment need to move down to the `catch (RuntimeException | UnsupportedEncodingException e)` line?